reliquery/metadata.py

```diff
@@ -517,6 +517,49 @@ def sync_relic_data(self, relic_data: RelicData) -> RelicData:
                 relic_data.relic_name, relic_data.relic_type, relic_data.storage_name
             )
 
+    def remove_old_relic_data(self, list_ids: List[int]) -> int:
+        """
+        Checks list of ids against database inorder to find any stale
+        relics rows and remove them.
+
+        param:
+            list_ids: integer - list of ids as integers
+
+        returns: int - number of rows removed
+        """
+        cur = self.conn.cursor()
```

Comment:
I think there is a way to do this in a single query. 
```
DELETE FROM  blah
WHERE blah_id NOT IN (1, 2, 3,); 
```

for instance. In general its important to try and make these queries "atomic."..